src/test/java/api/loans/DeclareLostAPITests.java

```diff
@@ -332,17 +332,12 @@ public void shouldNotAssignItemAnotherCostFeeIfAmountMissing(@Nullable Double it
   }
 
   @Test
-  @Parameters( {
-    "actualCost",
-    "null",
-    "someNewCostType"
-  })
-  public void shouldNotAssignFeeIfChargeTypeNotAnotherCost(@Nullable String chargeType) {
+  public void shouldNotAssignFeeIfChargeTypeIsActualCost() {
     final LostItemFeePolicyBuilder lostItemPolicy = lostItemFeePoliciesFixture
       .facultyStandardPolicy()
       .withName("Declared lost fee test policy")
       .doNotChargeProcessingFee()
-      .withChargeAmountItem(chargeType, 10.0);
+      .withChargeAmountItem("actualCost", 10.0);
```

Comment:
Could a method be introduced for actual costs, without a client needing to use `withChargeAmountItem` directly, to make the intent clearer. This would be similar to the set cost fee configuration.